### ISSUE TITLE
Toggle interactive rebase editor without aborting?

### ISSUE THREAD

**Issue by borekb** [eamodio/vscode-gitlens#1200]
The new interactive rebase editor is interesting but I need to build confidence in it – as many other people in the previous feedback issue https://github.com/eamodio/vscode-gitlens/issues/1153.

Since 11.0.2, it's possible to disable the new UI via global settings, which aborts the rebase and hides the UI "forever".

Would it be technically possible to implement a switch within a single rebase session? To me, an ideal behavior is that I can freely switch between the two _without aborting_ the rebase.

**Comment by eamodio** [comment]
Technically you can do this today with the vscode _Reopen Editor with_ command, but the problem is that vscode closes the document and then re-opens it. Which breaks the wait behavior, because once the document is closed git will start the rebase. I'm hoping we can get that fixed in vscode so that this could work properly.

I thought I had an existing issue opened for this, but I've now opened: https://github.com/microsoft/vscode/issues/111187

**Comment by MetRonnie** [comment]
I think this is implemented in 11.2.1? There's now a "Switch to text" button in the interative rebase UI

**Comment by eamodio** [comment]
Yeah, the experience is still not what I'd want (that vscode bug needs to be fixed) but the _Switch to Text_ button does provide this feature. It's just more fragile than I would like.
